project-handbook/content/docs/technical-committee.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Participating in the Technical Committee
+weight: 400
+---
+# Participating in the Technical Committee
+
+The Technical Committee is the working body of the OpenRail Assocation which acts as a bridge between the organization and the projects. It runs the incubation process and supports projects to work as high quality open source projects. The Technical Committee is a hands-on group which jointly does the overarching work necessary to support a healthy ecosystem of open source projects within the OpenRail Assocaition.
+
+Each project can send a representative into the Technical Committee. **As a project you need to name a person which gets invited to the weekly meetings of the Technical Committee** and has voting rights on decisions taken there. That said, almost all decisions are taken by consensus and those who do the actual work. We maintain a very collaborative culture there.
```

Comment:
```suggestion
Each project can send a representative into the Technical Committee. **As a project you need to name a person who gets invited to the weekly meetings of the Technical Committee** and has voting rights on decisions taken there. That said, almost all decisions are taken by consensus and those who do the actual work. We maintain a very collaborative culture there.
```
see which vs. who in the usage guide of the pronoun at:
https://www.merriam-webster.com/dictionary/which#dictionary-entry-2